### PR TITLE
Update TF Orb Documentation

### PR DESCRIPTION
**Pull request by mhbrooks**
Small documentation improvements.
Minor error case improved handling.

### CODE REVIEW

File: terraform/github.py [hunk 1]
```diff
@@ -76,6 +76,7 @@ def __init__(self, pr_url: str):
                     self._plan = match.group(1).strip()
                     self._status = match.group(2).strip()
                     return
+        raise Exception(f'No matching plan as comment on PR {pr_url} by username {github_username}.')
```
Comment (mhbrooks) [inline terraform/github.py]: Added this as currently if you set a github email as the USERNAME env, plan works okay but apply steps fail with just: 
```
None

Exited with code exit status 1
```

Comment (danielflookovo) [reply terraform/github.py]: I have a feeling this would break the apply step, as the comment won't exist yet

Comment (danielflookovo) [reply terraform/github.py]:
```suggestion

```

**Comment by mhbrooks** [comment]
Yes, good idea. I've not had chance to pick this up again, good to go ahead with the doc changes

**Pull request merged**